discovery-infra/test_infra/tools/tf_network_pool.json

```diff
@@ -1,20 +1,26 @@
 [
    {
       "machine_cidr":"192.168.126.0/24",
+      "machine_cidr6": "1001:db8::/120",
```

Comment:
You created a new `machine_cidr6` and don't override the existing `machine_cidr` because you want to support both IPv4 and IPv6 different cidrs?
Can't they have the same one?

Reply:
The machine_cidr6 is part of assets.  When nodes are created, single asset is allocated.  When it finishes, the asset is returned.  So asset has to include the relevant parameters for IPv4 and IPv6.